mimesis/enums.py

```diff
@@ -0,0 +1,115 @@
+from enum import Enum, EnumMeta
+from random import choice
+
+
+class AllowRandom(EnumMeta):
+    """This metaclass allow method get_random_item() which
+    equal to random field of enum for his subclasses.
+    """
+
+    def get_random_item(self):
```

Comment:
@sobolevn At this moment we have some issues related to this method. It's because we don't annotated this method:
```
mimesis/providers/units.py:325: error: Item "None" of "Optional[PrefixSign]" has no attribute "value"
mimesis/providers/cryptographic.py:36: error: Item "None" of "Optional[Algorithm]" has no attribute "value"
mimesis/providers/cryptographic.py:37: error: Item "None" of "Optional[Algorithm]" has no attribute "value"
mimesis/providers/code.py:61: error: Incompatible types in assignment (expression has type "str", variable has type "Optional[ISBNFormat]")
mimesis/providers/code.py:61: error: Item "None" of "Optional[ISBNFormat]" has no attribute "value"
# ...
# ...
```
How can we indicate in the annotations that the method returns self?

I try this:

```
def get_random_item(self) -> Enum:
    ...
```

But it did not help.

Reply:
@sobolevn I really need your help here. I don't know how to be with this problem.